src/components/LayerUI.tsx

```diff
@@ -211,7 +210,13 @@ const LibraryMenu = ({
   setAppState: React.Component<any, AppState>["setState"];
 }) => {
   const ref = useRef<HTMLDivElement | null>(null);
-  useOnClickOutside(ref, onClickOutside);
+  useOnClickOutside(ref, (ev) => {
+    // If click on the library icon, do nothing.
+    if ((ev.target as Element).closest(".ToolIcon_type_button__library")) {
+      return;
+    }
+    onClickOutside(ev);
+  });
```

Comment:
Just a naming tweak:

```suggestion
  useOnClickOutside(ref, (event) => {
    // If click on the library icon, do nothing.
    if ((event.target as Element).closest(".ToolIcon_type_button__library")) {
      return;
    }
    onClickOutside(event);
  });
```